labgrid/util/marker.py

```diff
@@ -2,5 +2,14 @@
 import string
 
 
-def gen_marker():
+BLOCKLIST = ("ERR", "FAIL", "WARN", "INFO", "DBG", "DEBUG")
+
+def _gen_marker():
     return ''.join(random.choice(string.ascii_uppercase) for i in range(10))
+
+def gen_marker():
+    """Generates marker not containing BLOCKLIST strings."""
+    while True:
+        marker = _gen_marker()
+        if not any(b in marker for b in BLOCKLIST):
```

Comment:
Is is necessary to do substring matching here? Why would a `XYZINFOABC` marker be a problem?

Reply:
Yes if we `grep INFO console_main_uut_console`

Reply:
If we want to avoid the delay we could also  remove "RID" from the random pool

Reply:
Ah, making a simple grep work sounds reasonable.

I'd prefer removing "RID" from the `choice()` parameter. Just make a local fixed list in the module and add a comment why these characters are missing.